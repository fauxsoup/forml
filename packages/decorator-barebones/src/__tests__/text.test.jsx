import Text from '../text';
import React from 'react';
import { render } from '@testing-library/react';

describe('renders', function() {
    test('its children', function() {
        const { container } = render(<Text><div id="test">test</div></Text>);

        expect(container).toMatchSnapshot();
        expect(container.querySelector('#test')).not.toBeNull();
    })
})
