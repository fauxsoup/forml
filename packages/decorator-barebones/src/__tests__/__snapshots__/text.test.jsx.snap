// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders its children 1`] = `
<div>
  <p>
    <div
      id="test"
    >
      test
    </div>
  </p>
</div>
`;
